NGitLab/Models/DetailedMergeStatus.cs

```diff
@@ -33,4 +33,7 @@ public enum DetailedMergeStatus
     NotOpen,
     [EnumMember(Value = "policies_denied")]
     PoliciesDenied,
+    // Undocumented member
+    [EnumMember(Value = "preparing")]
```

Comment:
The test case "Test_merge_request_api()" failed for version 16, because the enum value was null, but the string was "preparing":
Assert.That(mergeRequest.DetailedMergeStatus.EnumValue, Is.Not.Null);

PS: Someone else reported that behaviour as well: https://forum.gitlab.com/t/rest-api-get-single-mr-returns-undocumented-detailed-merge-status/98122